frontend/lib/src/components/elements/DeckGlJsonChart/styled-components.ts

```diff
@@ -46,6 +52,14 @@ export const StyledNavigationControlContainer = styled.div(({ theme }) => ({
       borderTopColor: theme.colors.secondaryBg,
     },
 
+    "&.mapboxgl-ctrl-icon:hover": {
+      // Lighten the background color on hover in dark mode (light mode works
+      // fine by default!)
+      backgroundColor: hasLightBackgroundColor(theme)
+        ? ""
+        : `color-mix(in srgb, ${theme.colors.bgColor} 90%, white 10%)`,
```

Comment:
nit: we already have some mixed colors based on the background in our theme? Would one of these colors also work here?

Reply:
E.g.
- bgMix
- darkenedBgMix100
- darkenedBgMix25
- darkenedBgMix15
- lightenedBg05

https://github.com/streamlit/streamlit/blob/82066be91838e7f321ec3770df71bf3f8894b23c/frontend/lib/src/theme/getColors.ts#L59

Reply:
I think the toolbar is pretty comparable, which uses `darkenedBgMix25` as the hover color.

Reply:
Great suggestion, updated to utilize that!

Reply:
Good callout @lukasmasuch !